test/util/test_cmaps.py

```diff
@@ -403,6 +403,7 @@ def test_create_colormap_from_config_color_entry_object(self):
                 "name": "my_cmap",
                 "type": "continuous",
                 "colors": [[0.0, "red"], [12.0, "#0000FF"], [24.0, [0, 1, 0, 0.3]]],
+                "labels": ["low", "medium", "high"],
```

Comment:
That's not correct - at least not what was specified:

` type ColorEntryTuple = [number, Color] | [number, Color, label];`

See also docs.